Makefile

```diff
@@ -3,20 +3,19 @@
 REGISTRY ?= quay.io
 REPOSITORY ?= $(REGISTRY)/kohlstechnology
 
-BUILD_COMMIT := $(shell ./scripts/build/get-build-commit.sh)
-BUILD_TIMESTAMP := $(shell ./scripts/build/get-build-timestamp.sh)
-BUILD_HOSTNAME := $(shell ./scripts/build/get-build-hostname.sh)
+COMMIT=`git rev-parse HEAD`
+BRANCH=`git symbolic-ref --short -q HEAD || echo HEAD`
+DATE=`date -u +%Y%m%d-%H:%M:%S`
+LDFLAG_LOCATION=github.com/KohlsTechnology/eunomia/version
```

Comment:
how about calling the variable `VERSION_PKG`  instead, to describe what is the meaning of the value? this is not really a LDFLAG by itself; also, I think we had a similar discussion in a different repo related to Raspberry Pi before ;P

Reply:
@akavel fixed in 928ee721be3f20a1a8b7f4282665e8fee5665643